salt/utils/event.py

```diff
@@ -584,8 +584,8 @@ def __del__(self):
         # shutdown-- where globals start going missing
         try:
             self.destroy()
-        except Exception as ex:
-            log.debug(ex)
+        except:
+            pass
```

Comment:
Since we have no way of knowing whether this is interpreter shutdown or not (meaning we don't know if loggers exist!) we'll just *pass* all these exceptions.